index.js

```diff
@@ -116,20 +122,28 @@ module.exports = function (themeopts) {
         });
       });
     }
-
     // return promise
     return new Promise(function (resolve, reject) {
+      const dimensions = {};
+      const getDimensions = (node) => {
+        if (node.height) {
```

Comment:
wouldn't this just get passed into the template?  Couldn't you just make the change there?